README.md

```diff
@@ -91,6 +91,11 @@ The following is a workflow formulated using the Texera GUI in a Web browser, wh
 * (9/2021) The Social Amplification and Attenuation of COVID-19 Risk Perception Shaping Mask Wearing Behavior: A Longitudinal Twitter Analysis, Suellen Hopfer, Emilia J. Fields, Yuwen Lu, Ganesh Ramakrishnan, Ted Grover, Quishi Bai, Yicong Huang, Chen Li, and Gloria Mark, in PLOS ONE 2021 [PDF](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0257428).
 * (4/2021) Why Do People Oppose Mask Wearing? A Comprehensive Analysis of US Tweets During the COVID-19 Pandemic, Lu He, Changyang He, Tera Leigh Reynolds, Qiushi Bai, Yicong Huang, Chen Li, Kai Zheng, and Yunan Chen, in JAMIA 2021 [PDF](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7989302/pdf/ocab047.pdf).
 
+## Education
+
+* [Data Science for All](https://ds4all.ics.uci.edu/)
+* [ICS 80](https://canvas.eee.uci.edu/courses/63639/pages/syllabus)
```

Comment:
Change "[ICS 80]" to "[ICS 80] (a course in spring 2024 to teach 42 UCI undergraduates, mostly not from CS, data science and AI/ML"

Reply:
Done.